aa-sdk/core/src/actions/smartAccount/waitForUserOperationReceipt.ts

```diff
@@ -0,0 +1,25 @@
+
+// TODO: fill in documentation
+
+import type {Client, Chain, Transport} from "viem";
+import type {WaitForUserOperationTxParameters} from "./types";
+import {UserOperationReceipt} from "../../types";
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.29.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../../types"
<!-- __reviewdog__:ChBlNTU3YTg2OTBjMzllYjczEgZyZGpzb24= -->